expirationd.lua

```diff
@@ -166,6 +166,8 @@ local function worker_loop(task)
             end
         end
 
+        --check if fiber was canceled in pcall
+        fiber.testcancel()
```

Comment:
Please add a space between `--` and message. Also please start a comment with a capital letter.

Reply:
Sorry, I missed the space again ((